bridge/opentracing/go.mod

```diff
@@ -5,17 +5,25 @@ go 1.18
 replace go.opentelemetry.io/otel => ../..
 
 require (
+	github.com/opentracing-contrib/go-grpc v0.0.0-20210225150812-73cb765af46e
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/stretchr/testify v1.8.1
 	go.opentelemetry.io/otel v1.13.0
 	go.opentelemetry.io/otel/trace v1.13.0
+	google.golang.org/grpc v1.52.3
```

Comment:
Are these dependencies pulled in only for testing?  If so, can they be moved to a `bridge/opentracing/test` module to minimize the impact on dependents that do not use gRPC and protobuf?

Reply:
good idea. But what do you do about code coverage in this case, is the CI setup to count usage from other modules as coverage?

Reply:
> good idea. But what do you do about code coverage in this case, is the CI setup to count usage from other modules as coverage?

Yes, it is setup to do that. 👍